proxy/src/schema/registry.rs

```diff
@@ -70,7 +81,7 @@ pub struct Applied {
 #[derive(Clone)]
 pub struct Registry {
     /// Registry client, whether an emulator or otherwise.
-    client: Client,
+    pub client: Client,
```

Comment:
Due to a testing complication it was either this or borrow the client with a lifetime, which I was too sick and tired to figure out. I'm pushing a commit shortly that changes to a static borrow.

The testing complication was that the client is needed in two places in the test suite: to pass to the Registry constructor, and to retrieve the project after registering the project. Since the Registry instance didn't use the client by reference, the client was locked to the instance.

Reply:
Actually I'm having some issues with the borrow checking while converting it to a borrow :grimacing: Maybe we can talk about this on Monday.

Reply:
Check 4c6556f for how to reuse the client, it derives `clone`.